way_to_home/static/src/components/routeForm/routeForm.js

```diff
@@ -20,9 +21,38 @@ class RouteSearchForm extends Component {
     }));
   }
 
-   getCurrentPosition = () => {
-     alert('get and write current position')
-   }
+   getCurrentPosition = (props) => {
+      if (!navigator.geolocation){
+        console.log("Geolocation is not supported by your browser")
+        return;
+      }
+      self = this
+      function success(position) {
+        let latitude  = position.coords.latitude;
+        let longitude = position.coords.longitude;
+         axios.get('https://nominatim.openstreetmap.org/reverse?format=jsonv2&lat='+latitude+'&lon='+longitude+'')
+            .then(function (response) {
+                let addr = response.data.address.city + ', ' + response.data.address.road;
+                if (response.data.address.house_number){
+                    addr += ", " + response.data.address.house_number;
+                }
+                if (props == 'A') {
+                    self.setPointA(addr)
+                }else{
+                    self.setPointB(addr)
+                }
+            })
+            .catch(function (error) {
+                console.log(error);
```

Comment:
Are you sure that we need this `console.log` ?

Reply:
yes

Reply:
In order for the user to enter the console, looked at the error and corrected it?

Reply:
ok